accurics-aws-tf-sample.tf

```diff
@@ -126,9 +126,21 @@ resource "aws_s3_bucket" "accurics-test-s3bucket1" {
   }
 
   tags = {
-    Name = format("%s-s3bucket1", var.acqaPrefix)
+    Name         = format("%s-s3bucket1", var.acqaPrefix)
     ACQAResource = "true"
-    Owner = "Accurics"
+    Owner        = "Accurics"
+  }
```

Comment:
**Accurics remediation**
 
**[HIGH]: S3 Buckets Server Side Encryption At Rest**
 
**Vulnerability**
Ensure that S3 Buckets have server side encryption at rest enabled to protect sensitive data.
 
**Remediation**
On the AWS S3 console, for each bucket that failed the rule, navigate to the Properties tab, and select Default encryption. Select an encryption method (AES-256 or AWS-KMS). Refer to: https://docs.aws.amazon.com/AmazonS3/latest/user-guide/default-bucket-encryption.html

source: accurics-app